lib/helpers/resolve-files.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-continue */
+
+const { promisify } = require('util');
+const { extname, join, isAbsolute, basename, sep, normalize } = require('path');
+const isGlob = require('is-glob');
+const glob = promisify(require('glob'));
+const {
+    removeTrailingSlash,
+    addLeadingSlash,
+    removeLeadingSlash,
+} = require('./path-slashes');
+const ResolvedFiles = require('../classes/resolved-files');
+
+/**
+ * Create a new path from a path string preceeding a glob or the whole path if no glob is found
+ *
+ * @param {string} path
+ * @returns {string} modified path
+ */
+const pathUntilGlob = (path) => {
+    const segments = (path || '').split('/');
```

Comment:
Will `.split('/');` work on windows? Should we be using [`path.sep`](https://nodejs.org/api/path.html#path_path_sep)?

Reply:
yeah should be path.sep indeed